sample/UrlGet/UrlGet.xcodeproj/project.pbxproj

```diff
@@ -44,6 +44,7 @@
 		BF_158572982416 /* Tests2.m in Sources */ = {isa = PBXBuildFile; fileRef = FR_447194245190 /* Tests2.m */; };
 		BF_161334189326 /* STPEmailAddressValidator.m in Sources */ = {isa = PBXBuildFile; fileRef = FR_820602908988 /* STPEmailAddressValidator.m */; };
 		BF_164244000470 /* GoogleAppIndexing.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = FR_353990188355 /* GoogleAppIndexing.framework */; };
+		"BF_164244000470-1" /* GoogleAppIndexing.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = FR_353990188355 /* GoogleAppIndexing.framework */; };
```

Comment:
This seems unintentional

Reply:
Hey Keith! You're right - I'm not sure what exactly caused this to pop up. It looks like now, `.framework` is included 2x in the `Link with libraries` phase for the Xcode built targets. My intention is to resolve this as part of wrapping up the upgrade ( and soon ) 😄